simexpal/launch/common.py

```diff
@@ -250,20 +263,47 @@ def compile_manifest(run):
 		'timeout': float(exp.info.timeout) if exp.info.timeout is not None else exp.info.timeout,
 		'environ': environ,
 		'output': exp.info.output,
+		'output_extensions': exp.info.output_extensions,
+		'stdout': exp.info.stdout,
```

Comment:
Why do we need `output` and `stdout`? Isn't one of these keys enough?

Reply:
I chose to keep the ``stdout`` key and removed the ``output`` key from ``RunManifest`` as it is not needed.